tests/unit/test_argumented_aggregates.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include "../../src/arithmetic/arithmetic_expression.h"
+#include "assert.h"
+
+void _test_ar_func(AR_ExpNode *root, SIValue expected) {
+    SIValue res = AR_EXP_Evaluate(root);
+
+    if(res.doubleval != expected.doubleval) {
+        printf("res.doubleval: %lf expected.doubleval: %lf\n", res.doubleval, expected.doubleval);
+    }
+
+    assert(res.doubleval == expected.doubleval);
+}
+
+void test_percentile_disc() {
+    // Percentiles to check
+    AR_ExpNode *zero = AR_EXP_NewConstOperandNode(SI_DoubleVal(0));
+    AR_ExpNode *dot_one = AR_EXP_NewConstOperandNode(SI_DoubleVal(0.1));
+    AR_ExpNode *half = AR_EXP_NewConstOperandNode(SI_DoubleVal(0.5));
+    AR_ExpNode *one = AR_EXP_NewConstOperandNode(SI_DoubleVal(1));
```

Comment:
It would be nicer if these were declared in increasing order 0, 0.1, 0.33, 0.5 and 1.